enterprise/api/v1/views.py

```diff
@@ -27,6 +28,20 @@ class EnterpriseReadOnlyModelViewSet(viewsets.ReadOnlyModelViewSet):
     throttle_classes = (ServiceUserThrottle,)
 
 
+class EnterpriseReadOnlyModelViewSet(EnterpriseModelViewSet, viewsets.ReadOnlyModelViewSet):
+    """
+    Base class for all read only Enterprise model view sets.
+    """
+    pass
+
+
+class EnterpriseReadWriteModelViewSet(EnterpriseModelViewSet, viewsets.ModelViewSet):
```

Comment:
@mattdrayer It looks like this PR overlaps heavily with that one. I didn't base this PR on it, so I accidentally rewrote some of the work in https://github.com/edx/ecommerce/pull/1127 as I completed this. :/

@mtyaka Do you think this PR fulfills the stuff in #36? If not, I will try to rebase this PR on top of that one and add in the new permission class.
Really sorry about the overlap I caused here @mtyaka - I should have checked more carefully that #36 had one of the API views needed for ENT-162, and then I should have rebased this PR on top of it before starting work on ENT-162.